src/style.css

```diff
@@ -1,3 +1,70 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
 body {
+  width: 100%;
+  margin: 0;
+}
+
+.img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.navbar {
+  line-height: 60px;
+}
+
+.list {
   margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+}
+
+.link {
+  color: #000;
+  text-decoration: none;
+  display: block;
+}
+
+.link:hover {
+  color: #00acdc;
```

Comment:
Similar to the previous comment, replace the hardcoded color #00acdc with a CSS variable to meet the task requirements.